app/models/article-model.js

```diff
@@ -0,0 +1,18 @@
+const db = require('../../db/connection')
+
+exports.fetchArticleById = (id) => {
+    return db.query(`SELECT * FROM articles WHERE article_id = ${id}`)
+    .then((response) => {
+        if(response.rowCount === 0){
+            return Promise.reject({status:404, msg:'id not found'})
+        }
+        return response.rows
```

Comment:
Here we want to be returning an object, rather than an array with an object inside (as we are only returning one article). Think about how we can just return the object inside. You will also need to amend your test to reflect this